.travis.yml

```diff
@@ -18,4 +18,4 @@ before_install:
  - cp appconfig.default.json appconfig.json
 
 script:
- - gulp
+ - ./node_modules/.bin/gulp
```

Comment:
maybe update readme. this change should be applied in the docker also

Reply:
Think it's running same gulp? Just being more specific.

Reply:
What